tests/test/wavm/test_ir_registry.cpp

```diff
@@ -51,7 +51,8 @@ TEST_CASE_METHOD(IRModuleCacheTestFixture, "Test main module caching", "[wasm]")
     // Sanity check the modules
     REQUIRE(!moduleRefA1.exports.empty());
     REQUIRE(!moduleRefB1.exports.empty());
-    REQUIRE(moduleRefA1.memories.defs[0].type.size.max == MAX_WASM_MEMORY_PAGES);
+    REQUIRE(moduleRefA1.memories.defs[0].type.size.max ==
```

Comment:
This was not formated properly, and the issue with the formatting checks failed to catch it.